kong/plugins/cors/api.lua

```diff
@@ -0,0 +1,3 @@
+return {
```

Comment:
@eshepelyuk A minor concern about this PR: the `api.lua` module name has a special meaning in plugins (it is used to specify customization to the generated Admin API endpoints). Please rename this module to something else to avoid namespace clashes.

Reply:
In fact, better not introduce such a new module at all. I understand that the intent is to not duplicate the list between the plugin's code and tests, but we actually would rather duplicate it. This avoids accidental changes to the list going unnoticed (since they need to be updated in the tests _and_ in the code). This is a pattern that this codebase has been following for a long time now.